python/config.py.in

```diff
@@ -0,0 +1,2 @@
+MLIR_C_RUNNER_UTILS = "@MLIR_DIR@/../../libmlir_c_runner_utils.so"
```

Comment:
This is not a good solution for Windows platform. Please try platform platform-independent solution.

Reply:
I agree with @ZhennanQin . Actually there is some examples in our repo for python binding `.py.in` to generate file names with OS-specific extension names. Also `@MLIR_DIR@/../../` seems fragile. I think we have already exposed this directory somewhere in `.py.in` ?

Reply:
my laptop has a problem now, I will fix this as soon as it returns to normal. I am trying to change it with `os.path` to implement platform-independent.
@Menooker we do have that exposed but it was in the test dir for lit config, I can not use that from here.  
I also noticed that we have something like the following in other `.py.in`, should change it to the solution with `os.path` ?
```
lit_config.load_config(config, "@PROJECT_SOURCE_DIR@/test/mlir/test/lit.cfg.py")
...
config.substitutions.append(("%gclibdir", config.gc_obj_root + "/lib/"))
...
```